exporter/src/main/scala/uk/gov/nationalarchives/consignmentexport/Main.scala

```diff
@@ -27,6 +27,7 @@ object Main extends CommandIOApp("tdr-consignment-export", "Exports tdr files in
         s3Files = S3Files(S3Utils(s3Async))
 
         data <- graphQlApi.getFiles(config, consignmentId)
+        _ <- IO.fromOption(data.headOption)(new Exception(s"No files found for consignment $consignmentId"))
```

Comment:
Really minor thing, but could this say something like "Consignment API returned no files for consignment $consignmentId"? If we ever hit this error, it will be helpful to know to check the DB/API first rather than S3 or EFS.